source/tests/CMakeLists.txt

```diff
@@ -251,15 +251,19 @@ add_timemory_google_test(type_trait_tests
 add_timemory_google_test(warning_tests
     SOURCES         warning_tests.cpp
     LINK_LIBRARIES  test-werror-flags
-                    test-debug-flags
                     timemory::timemory-core)
```

Comment:
```suggestion
                    $<$<NOT:$<BOOL:WIN32>>:test-debug-flags>
                    timemory::timemory-core)
```

Reply:
Why do you need the debug flags disabled on windows? I usually keep them there for better backtrace resolution when there are resolutions (but I don't have backtrace support for windows so it's fine if they are removed)

Reply:
Because on windows would not compile when linking against ``test-debug-flags``. 
I believe the cmake logic

```
add_library(test-debug-flags INTERFACE)
string(REPLACE " " ";" _FLAGS "${CMAKE_CXX_FLAGS_DEBUG}")
target_compile_options(test-debug-flags INTERFACE ${_FLAGS})
```
mixes the debug runtime into the release config build.
On windows you cannot mix the release and debug runtimes. 
If you want debug information, in the release build, you need to add ``/Zi`` to the compile step and ``/Debug`` to the link step.
Since I am not exactly sure what you are doing here or how to do that, I simply avoided linking against the ``test-debug-flags`` library on windows because I get a build error. (I do not remember if it was a compile-time or a link-time error.)